requirements.txt

```diff
@@ -15,7 +15,11 @@ python-dateutil==2.8.2
 vine==1.3.0
 amqp==2.6.1
 kombu>=4.6.11,<5
```

Comment:
According to this document kombu needs to be upgraded to version 5 here - https://docs.celeryq.dev/en/stable/history/whatsnew-5.0.html#kombu

I'm not really sure what kombu does.